readme.md

```diff
@@ -8,14 +8,12 @@ You might want to check out [XO](https://github.com/xojs/xo), which includes thi
 
 [**Propose or contribute a new rule ➡**](.github/contributing.md)
 
-
```

Comment:
restore blank lines in this file please

Reply:
More than one blank line is bad practice, especially since it doesn't affect the layout at all.